tajo-common/pom.xml

```diff
@@ -245,6 +249,12 @@
       <artifactId>powermock-api-mockito</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
```

Comment:
This can increase the size of the tajo client. How about moving it to other modules like tajo-core and tajo-plan?